lib/bolt/project.rb

```diff
@@ -73,7 +79,7 @@ def load_as_module?
 
     def name
       # If the project is in mymod/Boltdir/bolt-project.yaml, use mymod as the project name
-      dirname = @path.basename.to_s == 'Boltdir' ? @path.parent.basename.to_s : @path.basename.to_s
+      dirname = @path.basename.to_s.casecmp?(BOLTDIR_NAME) ? @path.parent.basename.to_s : @path.basename.to_s
```

Comment:
I think this check can just be `@type == 'embedded'`, which also better expresses the intent.